### PR TITLE
Update microsoft-365-guest-settings.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the setting where guests are to be added to a group.

Tested and it does not let even guests within the org to be added when the setting is OFF

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7166

File: microsoft-365/solutions/microsoft-365-guest-settings.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The Microsoft 365 admin center has organization-level settings for sharing and f
 | Setting | Default | Description |
 |:-----|:-----|:-----|
 |Let group members outside your organization access group content|On|When set to **On**, guests can access groups content; when set to **Off**, they can't. This setting should be **On** for any scenario where guests are interacting with Microsoft 365 Groups or Teams.|
-|Let group owners add people outside your organization to groups|On|When **On**, Owners of Microsoft 365 Groups or Teams can invite new guests to the group. When **Off**, owners can only invite guests who are already in the directory.|
+|Let group owners add people outside your organization to groups|On|When **On**, Owners of Microsoft 365 Groups or Teams can invite new guests to the group. When **Off**, they can't. This setting should be **On** for any scenario where guests are to be added to groups.|
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/microsoft-365-guest-settings.md]:
```suggestion
|Let group owners add people outside your organization to groups|On|When **On**, owners of Microsoft 365 Groups or Teams can invite new guests to the group. When **Off**, they can't. This setting should be **On** for any scenario where guests are to be added to groups.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**